wp-content/themes/engage/assets/font/cutfontawesome.css

```diff
@@ -0,0 +1,381 @@
+/*!
```

Comment:
I believe we had just removed Font Awesome. Was there a specific thing you needed to add? Ideally, we won't need these 1300 lines of CSS.

Reply:
I actually think that was just a misc file i added while testing something before the summer and forgot to delete, shouldn't be using font awesome anymore anywhere